UIKitDZ/UIKitDZ/AppetitViewController.swift

```diff
@@ -0,0 +1,129 @@
+//
+//  AppetitViewController.swift
+//  UIKitDZ
+//
+//  Created by Edvard on 30.06.2021.
+//
+
+import UIKit
+
+class AppetitViewController: UIViewController {
+    
+    var name = ""
+    var img = UIImage()
+    var pizzaImg = UIImageView(frame: CGRect(x: 100, y: 100, width: 250, height: 209))
+    var texasButton = UIButton()
+    let pizzaNamelabel = UILabel(frame: CGRect(x: 30, y: 30, width: 200, height: 21))
+    let switchLabel = UILabel(frame: CGRect(x: 30, y: 300, width: 200, height: 21))
+    let switchTwoLabel = UILabel(frame: CGRect(x: 30, y: 330, width: 200, height: 21))
+    let switchThreeLabel = UILabel(frame: CGRect(x: 30, y: 360, width: 200, height: 21))
+    let switchFourLabel = UILabel(frame: CGRect(x: 30, y: 390, width: 200, height: 21))
+    
+    let mySwitch = UISwitch()
+    let mySwitchTwo = UISwitch()
+    let mySwitchThree = UISwitch()
+    let mySwitchFour = UISwitch()
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = .white
+        labelNew()
+        
+        
+        pizzaImg.image = self.img
+        pizzaImg.backgroundColor = UIColor.clear
+        pizzaImg.tintColor = UIColor.gray
+        self.view.addSubview(pizzaImg)
+        
+        
+        self.texasButton = UIButton(type: .system)
+        self.texasButton.setTitle("Выбрать", for: .normal)
+        self.texasButton.frame = CGRect(x: 150, y: 585, width: 150, height: 40)
+        self.texasButton.addTarget(self, action: #selector(goToBayVC), for: .touchUpInside)
+        self.view.addSubview(self.texasButton)
+        
+        
+        self.mySwitch.frame = CGRect(x: 300, y: 300, width: 0, height: 0)
+        self.view.addSubview(self.mySwitch)
+        self.mySwitch.setOn(false, animated: true)
+        
+        self.mySwitchTwo.frame = CGRect(x: 300, y: 340, width: 0, height: 0)
+        self.view.addSubview(self.mySwitchTwo)
+        self.mySwitch.setOn(false, animated: true)
+        
+        self.mySwitchThree.frame = CGRect(x: 300, y: 380, width: 0, height: 0)
+        self.view.addSubview(self.mySwitchThree)
+        self.mySwitch.setOn(false, animated: true)
+        
+        self.mySwitchFour.frame = CGRect(x: 300, y: 420, width: 0, height: 0)
+        self.view.addSubview(self.mySwitchFour)
+        self.mySwitch.setOn(false, animated: true)
+        
+    }
+    
+    func labelNew () {
```

Comment:
```suggestion
    private func labelNew () {
```
тоже самое касается всех методов, если не вызывается из вне, ставится private